src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.figures;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, String name, int side) {
+        super(color, name);
+        this.side = side;
```

Comment:
The `name` parameter is being passed to the superclass constructor. Ensure that this is necessary, as the superclass might already handle the `name` property. If it's redundant, consider removing it.